v2/yarpcjson/inbound_test.go

```diff
@@ -41,6 +41,21 @@ type simpleResponse struct {
 	Success bool
 }
 
+func handleWithCodec(
+	ctx context.Context,
+	t *testing.T,
+	req *yarpc.Request,
+	reqBuf *yarpc.Buffer,
+	procedure yarpc.EncodingProcedure,
+) (*yarpc.Response, *yarpc.Buffer, error) {
+	p, err := yarpcrouter.EncodingToTransportProcedures([]yarpc.EncodingProcedure{
+		procedure,
+	})
+	require.NoError(t, err)
+
+	return p[0].HandlerSpec.Unary().Handle(ctx, req, reqBuf)
```

Comment:
Not actionable here, but based on how frequently we have to access the first element, i.e. `p[0]` it might be useful to add a `yarpcrouter.EncodingToTransportProcedure` (single) and have `yarpcrouter.EncodingToTransportProcedures` call this in its implementation.